### PR TITLE
Support Keplr Mobile

### PR DESCRIPTION
**Pull request by nnkken**
Not using `@likecoin/wallet-connect` since wanted to specify custom sign option, though seems still not supported by Keplr Mobile...
Only tested desktop website + Keplr Mobile signing, not yet tested on mobile website.
Deployed on https://likecoin-pluto.nnkken.dev/

### CODE REVIEW

**Comment by nnkken** [comment]
Current problems:

1. Since custom signing option is not supported on Keplr Mobile (or is it?), balance checking cannot be disabled, so Keplr Mobile will check the balance for transaction fee.
Say the transaction fee is 0.1 LIKE, then Keplr Mobile checks if the signing address has 0.1 LIKE, and forbid signing if balance is not enough.

2. The transaction fee may not be the one specified in transaction. Need `preferNoSetFee` signing options for solving that.

File: src/keplr/keplr.ts [hunk 1]
```diff
@@ -18,29 +23,39 @@ function onKeystoreChange(e: Event) {
   }
 }
 
+// TODO: what about wallet connect?
```
Comment (williamchong) [inline src/keplr/keplr.ts:26]: Is there a on disconnect event from wallet connect?

File: src/keplr/get-wc-keplr.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        case "sync":
+          return "BROADCAST_MODE_SYNC";
+        default:
+          return "BROADCAST_MODE_UNSPECIFIED";
+      }
+    })(),
+  };
+
+  const restInstance = Axios.create({
+    baseURL: KEPLR_CHAIN_INFO.rest,
```
Comment (williamchong) [inline src/keplr/get-wc-keplr.ts:36]: Should we add timeout?

File: src/keplr/keplr.ts (hunk 1 above)
Comment (nnkken) [reply src/keplr/keplr.ts:26]: Seems so, will check the docs (if any)

**Pull request merged**